stream.go

```diff
@@ -0,0 +1,99 @@
+package tarantool
+
+import (
+	"gopkg.in/vmihailenco/msgpack.v2"
+)
+
+type Stream struct {
+	Id   uint32
+	Conn *Connection
+}
+
+func fillBegin(enc *msgpack.Encoder) error {
+	return enc.EncodeMapLen(0)
```

Comment:
https://www.tarantool.io/en/doc/latest/dev_guide/internals/box_protocol/#iproto-begin-0x0e

It looks like you forgot about:
IPROTO_TIMEOUT and IPROTO_TXN_ISOLATION

Reply:
Oh, my bad, thank you for noticing:

https://github.com/tarantool/go-tarantool/blob/a027a592265114bd5a0e563c30a418d8671d809a/stream.go#L28-L52